### DISCUSSION TITLE
How to send a message with a person's username, without mentioning it?

### DISCUSSION THREAD

**Discussion by MrCheatEugene** [eternnoir/pyTelegramBotAPI#1696] [Q&A] [answered]
Hi. I am wrtiting a bot that mentions people sometimes. But, it also "pings" the person.

The issue here, if somebody will send like 100 commands.. person will get pinged 100 times, and mentioned in the message.
What I want is to mention the person, so if I click on username it will redirect me to DM that username, but without ping. Without notification. 

Is this possible? And if yes, how can I do that with PYTBA?

**Comment by coder2020official** [comment]
You can [mention](https://pytba.readthedocs.io/en/latest/util.html#telebot.util.user_link) the user using id, but he/she will get a ping.

**Reply by MrCheatEugene** [reply]
> You can [mention](https://pytba.readthedocs.io/en/latest/util.html#telebot.util.user_link) the user using id, but he/she will get a ping.

Thw question  was "how to mention, without ping?"
How I can mention person with ping I already know. 

Solution was to set parse mode to "html" and add a link like this:
`<a href="https://t.me/user">user</a>`
Now person doesn't get pinged, but mentioned.

**Comment by MrCheatEugene** [comment, marked as answer]
Solution was to set parse mode to "html" and add a link like this:
<a href="https://t.me/user">user</a>
Now person doesn't get pinged, but mentioned.

**Reply by coder2020official** [reply]
Good to see that you resolved it yourself
